src/bluetooth/TemperatureLogManager.js

```diff
@@ -0,0 +1,83 @@
+import moment from 'moment';
+
+// Default temperature log generator.
+const createTemperatureLog = ({ id, sensor, timestamp, temperature, logInterval }) => ({
+  id,
+  sensor,
+  timestamp: new Date(timestamp),
+  temperature,
+  logInterval,
+});
+
+export class TemperatureLogManager {
+  constructor(dbService, utils) {
+    this.db = dbService;
+    this.utils = utils;
+
+    this.createLogFunc = createTemperatureLog;
```

Comment:
Schemas differ slightly - allowing to setter inject a new creator for temperature logs.